### PR TITLE
add hall to site angular model

### PR DESCRIPTION
**Pull request by kds1660**
for now without api requests

### CODE REVIEW

File: src/js/Controllers/hallCtrl.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+angular.module('hall',[])
+    .controller('hallCtrl', ['$scope','hallService','cartService',
+        function ($scope,hallService,cartService) {
+           // hallService.setHall('Filarmonia');
+            hallService.setHall('Kulyka');
+        $scope.getUrl=function () {
```
Comment (Varkh) [inline src/js/Controllers/hallCtrl.js]: remove

File: src/js/Services/services.js [hunk 2]
```diff
@@ -100,4 +100,40 @@ theatreServices
             }
         }
     })
+
+    .factory('hallService', function() {
+        var hall='';
```
Comment (Varkh) [inline src/js/Services/services.js:105]: think about getting list of halls from back-end

File: src/js/Services/services.js [hunk 3]
```diff
@@ -100,4 +100,40 @@ theatreServices
...
+                sit: orders[2],
+                cost: 0,
+                area: orders[0]
+            });
+        },
+        getPlaceToCart: function () {
+            return order;
+        },
+        delPlaceFromCart: function (index) {
+            console.log(order[index],index)
```
Comment (Varkh) [inline src/js/Services/services.js]: no `console.log`

**Review by Varkh — CHANGES_REQUESTED** [review]

File: app/views/Performance/performance.html [hunk 4]
```diff
@@ -23,6 +23,12 @@ <h2 class="page-title">{{performance.title}}</h2>
                         <h3>{{performance.title}}</h3>
                         <p>{{date.time}}</p>
                     </div>
+                    <span>{{performanceEvents[$index].venue}}</span>
+                    <div class="block-description">
+                        <a ng-click="selectPerfomance($index)">
+                        <span >Придбати квиток</span>
```
Comment (Varkh) [inline app/views/Performance/performance.html]: As I understand, titles should be moved to file. As there are few languages. ex. `{{perfomance.performance.title}}`

File: src/js/Controllers/SinglePerfCtrl.js [hunk 5]
```diff
@@ -2,8 +2,8 @@
  * Created by Yana on 16.02.2015.
  */
 angular.module('singlePerf',['ui.bootstrap'])
-    .controller('SinglePerformanceCtrl', ['$scope', '$q', 'apiGet', '$stateParams', 'dateConvert', '$rootScope',
-        function ($scope, $q, apiGet, $stateParams, dateConvert, $rootScope) {
+    .controller('SinglePerformanceCtrl', ['$scope', '$q', 'apiGet', '$stateParams', 'dateConvert', '$rootScope','$state','perfomanceService',
+        function ($scope, $q, apiGet, $stateParams, dateConvert, $rootScope,$state,perfomanceService) {
```
Comment (Varkh) [inline src/js/Controllers/SinglePerfCtrl.js]: space ` ` before parameter

File: src/js/Directives/HallDirective.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+                        sitsData.sit=event.target.getAttribute('data-place');
+
+                        while (!sitsData.row&&!sitsData.sits) {
+                            nodeParent = nodeParent.parentNode;
+                            if (nodeParent.hasAttribute('data-row')) sitsData.row = nodeParent.getAttribute('data-row');
+                            if (nodeParent.hasAttribute('data-section')) sitsData.section = nodeParent.getAttribute('data-section');
+                        }
+                        scope.addToCart(sitsData)
+                    }
+
```
Comment (Varkh) [inline src/js/Directives/HallDirective.js]: remove unneeded new lines

**Review by Varkh — CHANGES_REQUESTED** [review]

**Comment by Varkh** [comment]
Update changed from DEV, as Merging is blocked
@kds1660

File: src/js/Directives/HallDirective.js (hunk 6 above)
Comment (kds1660) [reply src/js/Directives/HallDirective.js]: removed

File: src/js/Controllers/SinglePerfCtrl.js (hunk 5 above)
Comment (kds1660) [reply src/js/Controllers/SinglePerfCtrl.js]: added spaces

File: src/js/Services/services.js (hunk 2 above)
Comment (kds1660) [reply src/js/Services/services.js:105]: for now get from backend const halls and add in object
 var hallConst = {
                "Черкаська Філармонія": "Filarmonia",
                "Будинок культури ім. Кулика": "Kulyka",
                "Черкаський Театр": "",
                "Кінотеатр 'Салют'": "",
                "Черкаський міський Палац молоді": "",
                "Центр дитячої та юнацької творчості": ""
            };

Comment (kds1660) [reply src/js/Services/services.js:105]: for now dont have api, added from const, backend said, that they
will provide for a venue resource, but how i will convert names into html filename, if i receive halls...

File: app/views/Performance/performance.html (hunk 4 above)
Comment (kds1660) [reply app/views/Performance/performance.html]: this is strange translation at site, menu-ok, other -50/50. I think about it, i will try translate today, but venue from backend - ukrainian always

**Comment by Varkh** [comment]
@kds1660  Please, update from dev one more time

**Review by Varkh — APPROVED** [review]

**Pull request merged**